### PR TITLE
Add contact form portion 

### PR DESCRIPTION
**Pull request by beyk**
In this project, I and my partner have to work together and complete The contact form like Figma template

### CODE REVIEW

**Review by ReshyResh — CHANGES_REQUESTED** [review]
# Hi team, 

### Good job so far!
  - ✔Good use of FormSpree to collect user data.
  - ✔Correct HTML validation for the input


#### There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️

- [ ] Please make sure to stick to the Figma template for styling the form. Currently, the textarea doesn't match the design as it is smaller than it should, and the button doesn't have any interaction (hover, active..)

| Your design  | Figma design |
| ------------- | ------------- |
|   |   |




## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding! 👏👏👏

----

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles/main.css [hunk 1]
```diff
@@ -211,3 +211,115 @@ nav {
...
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  padding: 15px 0;
+  outline: none;
+  width: 93%;
+  height: 140px;
+  font-family: Poppins, Arial, Helvetica, sans-serif;
+  color: #b3bac5;
+  font-size: 15px;
+}
```
Comment (Whoistolu) [inline styles/main.css:288]: - [ ] [OPTIONAL] Hi team. you have done a great implementation, however, I see that the background color for your text area is `white` while the color itself is grey `#b3bac5`, this doesn't make the text input really accessible. Please change it to black or the color you gave the email and the name input.

Screenshot from 2022-01-19 15-26-40

**Review by Whoistolu — APPROVED** [review]
### Hi team,
@beyk and @vickymarz 

Your project is complete!  There is nothing else to say other than... It's time to merge.

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

_Check the inline comments under the review for sections to improve._

Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles/main.css (hunk 1 above)
Comment (beyk) [reply styles/main.css:288]: Thank you for your attention

**Pull request merged**